app/src/androidTest/java/ch/hikemate/app/ui/map/HikeDetailsScreenTest.kt

```diff
@@ -242,7 +249,8 @@ class HikeDetailScreenTest {
   @Test
   fun hikeDetails_showsCorrectDetailedHikesValues() {
     composeTestRule.setContent {
-      HikeDetails(detailedRoute = detailedRoute, savedHikesViewModel = savedHikesViewModel)
+      HikeDetails(
+          detailedRoute = detailedRoute, savedHikesViewModel = savedHikesViewModel, emptyList())
```

Comment:
LGTM, I was initially confused if we should not also test the ElevationGraph in this screen, but there are tests for the elevationgraph, so there is no need to retest its functionality here.